### DISCUSSION TITLE
DIA-NN library-free predicting RTs

### DISCUSSION THREAD

**Discussion by diabeginner01** [vdemichev/DiaNN#423] [Q&A]
Hi!
Thank you for providing this awesome software for the proteomics community! I am new to DIA and especially using it with predicted libraries. I am bit confused about an aspect related to using DIA-NN with library-free search. In the DIA-NN paper in the methods it is stated:

"When a sequence database is provided as input, DIA-NN generates a spectral library in silico. For this, DIA-NN can optionally use a fragmentation predictor (based on the approach introduced in MS Simulator) and a linear retention time predictor. **The predictors are trained using any spectral library supplied by the user**."

Do I understand correctly that if I do a library-free analysis and want DIA-NN to predict RTs as well, I would still need to provide a spectral library with RTs (coming from a run that uses the same gradient, length etc) so that the software trains itself for predicting the RTs for my specific chromatographic conditions? And if I don't provide such a spectral library for RT prediction, then are the results any way worse than when having one? I'm asking this as I seem to be able to analyze my DIA run just fine with library-free and providing only the fasta db, but am wondering if the results would be any better if additionally providing a spec lib for RT prediction.

Thanks for any help!

**Comment by vdemichev** [comment]
Hi, please rely solely on the DIA-NN documentation here on github, not the DIA-NN paper. The paper describes a very old version of DIA-NN, many things have changed since then.

Best,
Vadim
